packages/tokens/package.json

```diff
@@ -29,6 +29,9 @@
 		"./themes.css": "./dist/themes.css",
 		"./index.css": "./dist/index.css",
 		"./media-queries.css": "./dist/media-queries.css",
```

Comment:
I considered just updating the existing "virtual" exports above, but that would be a breaking change (open to feedback though!)